docs/tutorials/vagrant-overview.rst

```diff
@@ -0,0 +1,14 @@
+Overview
+========
+
+There are two kinds of environments for RackHD, demo and development environment. 
```

Comment:
How to use vagrant to set up demo and development environment of RackHD is introduced

how about it ?